pkg/oauth/apiserver/registry/tokenreviews/etcd.go

```diff
@@ -0,0 +1,55 @@
+package etcd
+
+import (
+	"context"
+	"fmt"
+
+	kauthenticationv1 "k8s.io/api/authentication/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/authentication/authenticator"
+	"k8s.io/apiserver/pkg/registry/rest"
+	kauthinternal "k8s.io/kubernetes/pkg/apis/authentication"
+	kauthv1internal "k8s.io/kubernetes/pkg/apis/authentication/v1"
+	"k8s.io/kubernetes/pkg/registry/authentication/tokenreview"
```

Comment:
is this dependency new?

Reply:
yes, it bears the upstream implementation to handle the endpoint